gamatrix-gog.py

```diff
@@ -468,24 +434,22 @@ def parse_cmdline(argv: List[str]) -> Any:
     if "mode" in config and config["mode"] == "server":
         # Start Flask to run in server mode until killed
         app.config["UPLOAD_FOLDER"] = config["db_path"]
-        app.config["MAX_CONTENT_LENGTH"] = UPLOAD_MAX_SIZE
+        app.config["MAX_CONTENT_LENGTH"] = constants.UPLOAD_MAX_SIZE
         app.run(host=config["interface"], port=config["port"])
         sys.exit(0)
 
-    if args.userid is None:
+    user_ids_to_compare = opts.get("--userid", [])
+    if user_ids_to_compare == []:
         user_ids_to_compare = [u for u in config["users"].keys()]
-    else:
-        user_ids_to_compare = args.userid
 
     # init_opts() is meant for server mode; any CLI options that are also
     # web UI options need to be overridden
-    opts = init_opts()
-    opts["include_single_player"] = args.include_single_player
-    for userid in user_ids_to_compare:
-        opts["user_ids_to_compare"][userid] = config["users"][userid]
-
+    web_opts = init_opts()
+    web_opts["include_single_player"] = opts.get("--include-single-player", False)
+    web_opts["user_ids_to_compare"] = user_ids_to_compare
```

Comment:
Are you sure this works? I think we still need the `for` loop on old lines 484-485 to add the data under each userid.

Reply:
I agree, you caught another one. :thinking: If only there was some way we could _test for such stuff_.